Shear/App.swift

```diff
@@ -12,6 +12,12 @@ import Cocoa
 class App {
     let statusMenu = StatusMenu()
     
+    let history: HistoryType
+    
+    init(history: HistoryType) {
+        self.history = history
+    }
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/895e50bbb85826ea77037e4a1c1e5468d960c4fc/Shear/App.swift#L20"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>App.swift:20</code></del></td>
    </tr>
  </tbody>
</table>